src/commons/application/actions/SessionActions.ts

```diff
@@ -152,6 +152,7 @@ export const setAdminPanelCourseRegistrations = (
 
 export const setGoogleUser = (user?: string) => action(SET_GOOGLE_USER, user);
 
+// TODO: Does not seem to be used anywhere, perhaps we can remove this?
 export const setGitHubAssessment = (missionRepoData: MissionRepoData) =>
   action(SET_GITHUB_ASSESSMENT, missionRepoData);
```

Comment:
This action seems unused, shall we remove the action and relevant reducers/types?

Reply:
The development and usage of GitHub Assessments does not seem to be very active at the moment, I think we can go ahead and remove this.